README.md

```diff
@@ -1,5 +1,8 @@
 # polly-jest-presets
 
+![build](https://img.shields.io/circleci/build/github/spotify/polly-jest-presets.svg)
+![version](https://img.shields.io/github/package-json/v/spotify/polly-jest-presets.svg)
```

Comment:
these will just link to the image if you click them
should we add another link so you get taken to circle-ci/npm if you click?
`[![version](badge-url)](click-url)`

Reply:
yep good idea. should the version badge send you to the npm page? or the npm version page? or something else?